src/diffusers/pipelines/t2i_adapter/pipeline_stable_diffusion_adapter.py

```diff
@@ -727,9 +727,14 @@ def __call__(
         extra_step_kwargs = self.prepare_extra_step_kwargs(generator, eta)
 
         # 7. Denoising loop
-        adapter_state = self.adapter(adapter_input)
-        for k, v in enumerate(adapter_state):
-            adapter_state[k] = v * adapter_conditioning_scale
+        if isinstance(self.adapter, MultiAdapter):
+            adapter_state = self.adapter(adapter_input, adapter_conditioning_scale)
```

Comment:
I'm a bit confused here, is `adapter_conditioning_scale` supposed to be similar to `adapter_weights` ? We are passing`adapter_weights` to `MultiAdapter` here 
https://huggingface.slack.com/archives/C05PH2CH4N5/p1693571995003459

Reply:
But, now we have 2 factors that are the same manner, I m also confuse if we do that way.

Reply:
ok, yes the change in the PR is correct. The multi adapter applies the additional multiplication internally to the module and the regular adapter's results are multiplied in the pipeline. It's a bit non-uniform between the two but I think is ok